community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/IndexStoreView.java

```diff
@@ -44,10 +49,17 @@
      * @return a {@link StoreScan} to start and to stop the scan.
      */
     <FAILURE extends Exception> StoreScan<FAILURE> visitNodes( int[] labelIds, int[] propertyKeyIds,
```

Comment:
@MishaDemianenko specifically requested to introduce it, instead of passing in `null` to the other one. You see, there's more work to do in the store scan if the label visitor `!= null`. I'm fine with removing it though, it's just conflicting comments.